pkg/oneinch/fusionorder/address.go

```diff
@@ -0,0 +1,19 @@
+package fusionorder
+
+import "github.com/ethereum/go-ethereum/common"
+
+const (
+	addressHalfLength = common.AddressLength / 2
+)
+
+type AddressHalf [addressHalfLength]byte
+
+func HalfAddressFromAddress(a common.Address) AddressHalf {
```

Comment:
mostly we never modify an address, I think get a slice of bytes will save some allocation.

Reply:
Because `AddressHalf` is a static slice of bytes, same as `common.Address`. So I decided still keep it.